scripts/covid_tracking_project/historic_state_data/preprocess_csv.py

```diff
@@ -63,7 +63,7 @@
 variableMeasured: dcs:{stat_var}
 measurementMethod: dcs:CovidTrackingProject
 observationAbout: C:COVIDTracking_States->GeoId
-observationDate: C:COVIDTracking_States->Dates
+observationDate: C:COVIDTracking_States->Date
```

Comment:
Oof, good catch. How did this work before?

Reply:
No worry about this, I am thinking this is because I unintentionally add a 's' when using 'ctrl + s' to save file. The tmcf in prod is the correct one~~